src/python/WMArchive/Schemas/FWJRProduction.py

```diff
@@ -74,42 +73,54 @@
                          'validStatus': 'PRODUCTION',
                          "SEName": "srm-cms.cern.ch",
-                         "PNN": "T2_CERN_CH",
+                         "PNN": "T2_CH_CERN",
                          "GUID": '',
                          "StageOutCommand": "srmv2-lcg"}],
-              'performance': {#"multicore": {},
-                  "storage": {
-                    "readAveragekB": 77.8474891246,
-                    "readCachePercentageOps": 0.0,
-                    'readMBSec': 0.0438598972596,
-                    'readMaxMSec': 4832.84,
-                    'readNumOps': 97620.0,
-                    'readPercentageOps': 1.00032780168,
-                    'readTotalMB': 7423.792,    
-                    'readTotalSecs': 0.0,
-                    'writeTotalMB': 357.624,
-                    'writeTotalSecs': 575158.0},
-                    "memory": {
-                       "PeakValueRss": 0.0,
-                       "PeakValueVsize": 0.0
-                    },
-                   "cpu": {
-                       "TotalJobCPU": 0.39894,
-                       "AvgEventCPU": -2.0, # for ("-nan")
-                       "MaxEventCPU": 0.0,
-                       "AvgEventTime": -1.0, # for ("inf")
-                       "MinEventCPU": 0.0,
-                       "TotalEventCPU": 0.0,
-                       "TotalJobTime": 26.4577,
-                       "MinEventTime": 0.0,
-                       "MaxEventTime": 0.0
-                   }},
+              'performance': {'cpu': [{'pName': 'TotalJobCPU',
+                                        'value': 159.08},
+                                       {'pName': 'AvgEventCPU',
+                                        'value': 0.0543207},
+                                       {'pName': 'MaxEventCPU',
+                                        'value': 2.01969},
+                                       {'pName': 'AvgEventTime',
+                                        'value': 0.0915972},
+                                       {'pName': 'MinEventCPU',
+                                        'value': 0.0},
+                                       {'pName': 'TotalEventCPU',
+                                        'value': 128.957},
+                                       {'pName': 'TotalJobTime',
+                                        'value': 217.452},
+                                       {'pName': 'MinEventTime',
+                                        'value': 0.000827074},
+                                       {'pName': 'MaxEventTime',
+                                        'value': 3.28857}],
+                              'memory': [{'pName': 'PeakValueRss',
+                                           'value': 692.926},
+                                          {'pName': 'PeakValueVsize',
+                                           'value': 952.109}],
+                              'multicore': [],
```

Comment:
Seangchan,
from my side the change is ok, I'm awaiting for you to merge, then I can build
RPMs, setup WMArchive and we can proceed with integration tests for WMArchive.
Thanks,
Valentin.

On  0, ticoann <notifications@github.com> wrote:
> > +                                        'value': 0.0915972},
> > +                                       {'pName': 'MinEventCPU',
> > +                                        'value': 0.0},
> > +                                       {'pName': 'TotalEventCPU',
> > +                                        'value': 128.957},
> > +                                       {'pName': 'TotalJobTime',
> > +                                        'value': 217.452},
> > +                                       {'pName': 'MinEventTime',
> > +                                        'value': 0.000827074},
> > +                                       {'pName': 'MaxEventTime',
> > +                                        'value': 3.28857}],
> > +                              'memory': [{'pName': 'PeakValueRss',
> > +                                           'value': 692.926},
> > +                                          {'pName': 'PeakValueVsize',
> > +                                           'value': 952.109}],
> > +                              'multicore': [],
> 
> Hi Valentin, It is also for the what actual names and values might be in the schema as well as schema generation. 
> But if it cause the problem, I can simplify as you suggested. Let me know.
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMArchive/pull/158/files/3172ba28998f9edf08a1bb244c2e0ac2e9b612cb#r59609928